core/src/main/java/io/questdb/griffin/engine/table/FilterOnValuesRecordCursorFactory.java

```diff
@@ -188,7 +204,7 @@ protected RecordCursor getCursorInstance(DataFrameCursor dataFrameCursor, SqlExe
         for (int i = 0, n = cursorFactories.size(); i < n; i++) {
             cursorFactories.getQuick(i).getFunction().init(dataFrameCursor, sqlExecutionContext);
         }
-
+        //potentially changes factory order so explain will show plan  different from actual  
```

Comment:
i don't understand the comment, could you rephrase?

Reply:
It means that sorting cursorFactories here can lead to slightly different execution plan compared to one produced by explain . That's because explain doesn't call getCursorInstance().

Reply:
cool, could we update the comment with this description pls? :)